Lib/compositor/font.py

```diff
@@ -6,6 +6,17 @@
 from compositor.error import CompositorError
 
 
+def _nameToString(nameRecord):
+    encoding = nameRecord.getEncoding()
+    try:
+        # python 3
+        ret = nameRecord.string.decode(encoding)
```

Comment:
How does that one behave on Python 3? I was looking to return the canonic str representation for 2 and 3. 
Thinking about it, maybe I should do toUnicode() and then tostr() on the result. Does that sound good?